### DISCUSSION TITLE
Export themes to different repo

### DISCUSSION THREAD

**Discussion by SharakPL** [JanDeDobbeleer/oh-my-posh#1128] [Ideas]
This frequent version bump just because a theme is added/fixed is somewhat annoying. I like to be up to date but what's the point of updating if app itself hasn't changed. I have my own theme so all other themes are irrelevant to me. Package is already quite large and imho it's pointless to fill it up more with themes.

So the idea is to make oh-my-posh completely free of themes. You would simply `Install-Module oh-my-posh` and then installed your theme from different repository eg. `omp install fish` and updated Powershell profile.
If anyone would want to check them all then `omp install` would be enough. The same goes for updating: `omp update fish` or `omp update` for all.

**Comment by JanDeDobbeleer** [comment]
We discussed this before, but truth be told themes are not the issue here. We push a lot more non-theme changes and basically have a new release daily which almost always contains code changes. Easy calculation: 113 theme commits in a total of 875 (13%). Size is also limited to 373KB so that's not an issue either.

**Reply by JanDeDobbeleer** [reply]
@SharakPL maybe [this](https://github.com/JanDeDobbeleer/oh-my-posh/pull/1130) is enough to remove probable overhead.

**Reply by SharakPL** [reply]
one small step for mankind 😀 though I feel much more can be done...

**Reply by JanDeDobbeleer** [reply]
we do accept PR's...
